pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -1676,6 +1719,48 @@ def create_model(df_sets, df_parameters, default={}):
             units=model.model_units["pipe_cost_capacity"],
             doc="Pipeline construction/expansion capital cost for selected increment [currency/volume/time]",
         )
+
+    _df_parameters = {}
```

Comment:
I suggest being more specific with this variable name. Something along the lines _elevation_adjusted_piping_cost?

Reply:
i am not sure if i should rename it, because the variable is the same whether we use hydraulics or not

Reply:
I agree that a more descriptive variable name would be helpful here. I think I would probably just go with something like `pipeline_operational_cost`. The variable is used regardless of whether or not hydraulics are being used, but in any scenario, it's being used to hold the pipeline operational cost values.